### ISSUE TITLE
Toggle clouds via HUD

### ISSUE THREAD

**Issue by tuxayo** [dragonfireclient/dragonfireclient#65]
## Problem

Since this fork is often used for flying, clouds are quickly a bother. Especially when wanting to see a very large area when coupled with the following in minetest.conf
client_mapblock_limit = -1
client_unload_unused_data_timeout = 6000000

Or just when building quite high.

## Solutions

Implement a toggle, in render for example. 

## Alternatives

enable_clouds = false in minetest.conf

But one has to restart the game to apply the changes. And having clouds is nicer while on the ground or low the rest of the time.

**Comment by corarona** [comment]
> But one has to restart the game to apply the changes. And having clouds is nicer while on the ground or low the rest of the time.

yeah but prob not worth going through the hassle of applying that "live"
